faunadb/serialization_test.go

```diff
@@ -173,6 +173,26 @@ func TestSerializeStructWithTags(t *testing.T) {
 	)
 }
 
+func TestSerializeStructWithOmitEmptyTags(t *testing.T) {
+	type user struct {
+		Name          string   `fauna:"name,omitempty"`
+		LastName      string   `fauna:"last_name,omitempty"`
+		Age           int      `fauna:"age,omitempty"`
+		Siblings      int      `fauna:"siblings,omitempty"`
+		Height        float64  `fauna:"height,omitempty"`
+		Weight        float64  `fauna:"weight,omitempty"`
+		HeightPointer *float64 `fauna:"height_pointer,omitempty"`
+		WeightPointer *float64 `fauna:"weight_pointer,omitempty"`
```

Comment:
can you enhance this struct to make it clear that the empty values will not be serialized, you don't need to a "real" user struct, I'm more interested to see the all the combinations of data types you can omit, something along in the lines
```go
	type omitStruct struct {
		String string `fauna:str,omitempty`,
		Int *int `fauna:int,omitempty`
		....
	}

	assertJSON(t,
 		Obj{"data": omitStruct{nil, nil, ....}},
 		`{"object":{"data":{"object":{}}}}`,
 	)

	assertJSON(t,
 		Obj{"data": omitStruct{"String", nil, ....}},
 		`{"object":{"data":{"object":{"str":"String"}}}}`,
 	)

 	//more assertions
```

Reply:
certainly, ill update the tests to make it more clear how it handles different empty values.